xml/System.Security.Cryptography/ECDsa.xml

```diff
@@ -1432,13 +1432,22 @@ The algorithm-specific key import failed.
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" Index="3" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="data">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="hashAlgorithm">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="data">The data to be hashed.</param>
+        <param name="destination">The buffer to receive the hash value.</param>
+        <param name="hashAlgorithm">The algorithm to use to hash the data.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes written into <paramref name="destination"/>. This parameter is treated as uninitialized.</param>
+        <summary>Attempts to compute the hash value of the specified read-only span of bytes into the provided destination by using the specified hashing algorithm.</summary>
+        <returns><see langword="false"/> if <paramref name="destination"/> is not long enough to receive the hash value.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+The default implementation of this method is to call <xref:System.Security.Cryptography.ECDsa.HashData(System.Byte[], System.Int32, System.Int32, System.Security.Cryptography.HashAlgorithmName)> and copy the result to `destination`.
```

Comment:
The build [had a warning ](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C11%5C25%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C201911251733433084-3564%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=CFGX0qjAqjq356%2FYCreWkHb53MV0l24SaaGm0YaZBmc%3D&st=2019-11-25T18%3A07%3A32Z&se=2019-12-26T18%3A12%3A32Z&sp=r) stating that this HashData DocId does not exist. You can see the problem [in the preview of this remarks item](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.ecdsa.tryhashdata?view=netcore-3.0&branch=pr-en-us-3564&viewFallbackFrom=netframework-4.8).

There are two DocIds available:

`M:System.Security.Cryptography.ECDsa.HashData(System.IO.Stream,System.Security.Cryptography.HashAlgorithmName)`
`M:System.Security.Cryptography.ECDsa.HashData(System.Byte[],System.Int32,System.Int32,System.Security.Cryptography.HashAlgorithmName)`

I assume you wanted to use the second one, so here's the suggestion:

```suggestion
The default implementation of this method is to call <xref:System.Security.Cryptography.ECDsa.HashData(System.Byte[],System.Int32,System.Int32,System.Security.Cryptography.HashAlgorithmName)> and copy the result to `destination`.
```